src/optimizer/query_to_operator_transformer.cpp

```diff
@@ -45,18 +45,17 @@ void QueryToOperatorTransformer::Visit(const parser::SelectStatement *op) {
 
   if (op->where_clause != nullptr) {
     SingleTablePredicates where_predicates;
-    util::ExtractPredicates(op->where_clause, where_predicates, join_predicates_);
```

Comment:
This looks like just formatting changes. What logic did you change in this file?